rest-api/src/main/java/io/quarkus/qe/services/RepositoryService.java

```diff
@@ -0,0 +1,36 @@
+package io.quarkus.qe.services;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import javax.transaction.Transactional;
+import javax.ws.rs.NotFoundException;
+
+import org.eclipse.microprofile.reactive.messaging.Channel;
+import org.eclipse.microprofile.reactive.messaging.Emitter;
+import io.quarkus.qe.configuration.Channels;
+import io.quarkus.qe.data.RepositoryEntity;
+import io.quarkus.qe.data.marshallers.RepositoryMarshaller;
+import io.quarkus.qe.model.Repository;
+import io.quarkus.qe.model.requests.NewRepositoryRequest;
+
+@ApplicationScoped
+public class RepositoryService {
+
+    @Inject
+    @Channel(Channels.NEW_REPOSITORY)
+    Emitter<NewRepositoryRequest> newRepositoryRequestEmitter;
+
+    @Transactional
```

Comment:
I am not sure if is required at the service level.

Reply:
Well spotted! Removed.